src/components/categories/Categories.js

```diff
@@ -20,15 +21,18 @@ const Categories = props => {
 
           <div className="categories-tab">
             {collection.categories.map(category => (
-              <Link to={`/products/${collection.param}/${category.param}`}>
+              <Link
+                key={uuid()}
+                to={`/products/${collection.param}/${category.param}`}
+              >
                 <div key={category.id} className="image-card shadow">
                   <img className="image" src={category.imageRef} alt="" />
                   <p className="">{category.categoryName}</p>
```

Comment:
Remove className if u r not adding any classes to the tag

Reply:
my bad. I will remove that.